### ISSUE TITLE
Images causing IllegalArgumentException

### ISSUE THREAD

**Issue by GabrielSiq** [delas/plg#9]
I recently downloaded PLG as it would be very helpful for a project but couldn't run it. At first, I tried the .exe version, which prompted me to download java (already installed). I then, tried the JAR, which crashed before opening, accusing IllegalArgumentException: input == null in MainToolbar.java line 24.

After building the JAR from the source code myself, I noticed that all lines of code that load/set icons/images are causing this error. 

Can it maybe be an issue of the JRE version I am running? (most recent)

Thanks!

**Comment by GabrielSiq** [comment]
It works with Java 8. It was probably an issue specific to 10.

**Comment by delas** [comment]
Thanks for reporting the problem, Gabriel, I'll check this soon.

**Comment by GabrielSiq** [comment]
Just updating now that I understood it better. 

The issue I originally reported was fixed by the latest commit from January, but since I had downloaded the latest release, it wasn't included. After downloading the current master and building the JAR, I am able to run the program without the original issues. Other issues arise, probably because of the Java 10 I was using, so I just installed Java 8 and now it runs perfectly :)
